Feature/call contracts

Sotatek-DucPham opened the pull request.

thith commented:
@Sotatek-DucPham 
1. If the contract address does not exist, or if it is not a contract (does not have "src" state), just throw
2. Sender should be enforced by runtime, don't let contract decide
3. Don't rely on getFuncNames, try use Proxy instead. If Proxy does not help, then we may have to accept syntax like this:
```js
contract.call('methodName", params);
```

Sotatek-DucPham commented:
> @Sotatek-DucPham
> 
> 1. If the contract address does not exist, or if it is not a contract (does not have "src" state), just throw
> 2. Sender should be enforced by runtime, don't let contract decide
> 3. Don't rely on getFuncNames, try use Proxy instead. If Proxy does not help, then we may have to accept syntax like this:
> 
> ```js
> contract.call('methodName', params);
> ```

1. It is handled in `Worker` function
2. Fixed it
3. Fixed it

thith commented:
#2
